x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -92,26 +92,26 @@
         "throttle_time": {
           "properties": {
             "min": {
-              "type": "long"
+              "type": "keyword"
             },
             "avg": {
-              "type": "float"
+              "type": "keyword"
             },
             "max": {
-              "type": "long"
+              "type": "keyword"
             }
           }
         },
         "schedule_time": {
           "properties": {
             "min": {
-              "type": "long"
+              "type": "keyword"
```

Comment:
I think we should fix the collectors not the schema. it does not make sense to have these numeric values as keywords